crawl-ref/source/dat/des/variable/mini_features.des

```diff
@@ -2758,6 +2758,101 @@ MAP
 ..............
 ENDMAP
 
+NAME: chequers_decorated_corner
+TAGS: decor extra
+DEPTH: D, Depths
+MAP
+xxxxxxxxxx
+x.......G@
+x.TG.....@
+x.Gxxxxxxx
+x..x
+xG.x
+x@@x
+ENDMAP
+
+NAME: chequers_jeans
+TAGS: decor extra
+DEPTH: D, Depths
+SHUFFLE: G$
```

Comment:
I don't see a `$` to shuffle with here. Do you mean this?

    SUBST: G = G$

Reply:
Alex was probably just trying to have a 50% chance to place nothing instead of a statue, in which case maybe
`SUBST: G = G.`
would be more clear. If the `$` was trying to place something else, then this glyph needs to be defined.

Reply:
oops. the intention was to place a statue, gold or nothing. So this needs work